### DISCUSSION TITLE
Feature request: Easier use with Bazel-built Angular apps

### DISCUSSION THREAD

**Discussion by pshields** [angular/angularfire#1431] [Ideas]
Hi,

It would be great to use Firebase with Angular apps built with [Bazel](https://bazel.build/). Using Angular with Bazel and Closure to bundle Angular apps is [becoming increasingly feasible](https://github.com/angular/angular/issues/19058); Angular itself can be built from Bazel, work is in progress to [build Angular Material with Bazel](https://github.com/angular/material2/issues/8369), and [RxJS has also already made progress becoming more easily consumed from Bazel-built apps](https://github.com/ReactiveX/rxjs/pull/3168).

I tried adding angularfire2 to my Angular/Bazel app, but ran into [issues getting it working with the `ts_devserver` build rule](https://github.com/alexeagle/angular-bazel-example/issues/56). It looks like I may be able to do some wrangling to load it using SystemJS. But as alexeagle@ mentioned in https://github.com/alexeagle/angular-bazel-example/issues/56, another option worth considering is for angularfire2 to include BUILD files in its distribution and be able to be built by Bazel in addition to the current Rollup build.

Any progress towards making it easier to use angularfire2 from Bazel-built Angular apps would be appreciated!

**Comment by alexeagle** [comment]
Chatted with @davideast, I think we'll work together on this next month. It would be the same work for making angularfire available internally at google.

**Comment by hiepxanh** [comment]
hello, is there something new?

**Comment by jamesdaniels** [comment]
Let's do this!!!!

**Comment by alexeagle** [comment]
My work-in-progress is on this branch:
https://github.com/alexeagle/angularfire2/tree/bazel
if anyone has time to pick this up and get the tests passing, that would be awesome! otherwise maybe David and I can get to it later...
